keyboards/lattice60/rules.mk

```diff
@@ -0,0 +1,70 @@
+# Copyright 2019 Ethan Durrant (emdarcher)
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+# MCU name
+MCU = atmega328p
+
+# Bootloader selection
+#   Teensy       halfkay
+#   Pro Micro    caterina
+#   Atmel DFU    atmel-dfu
+#   LUFA DFU     lufa-dfu
+#   QMK DFU      qmk-dfu
+#   ATmega32A    bootloadHID
+#   ATmega328P   USBasp
+BOOTLOADER = USBasp
+
+
+# set the processor frequency to 12MHz
+F_CPU = 12000000
+
+# disable the debugging code
+OPT_DEFS = -DDEBUG_LEVEL=0
+
+# custom command for programming via USBasp for the 
+# ATmega328p running USBasploader
+# can be flashed with the following QMK make parameters:
+# make lattice60:default:program
+PROGRAM_CMD = avrdude -c usbasp -p atmega328p -U flash:w:$(BUILD_DIR)/$(TARGET).hex
```

Comment:
This shouldn't be needed.  There is the `USBasp` target, which is called by the `flash` target. 
```suggestion
```